pkg/models/terminal/terminal.go

```diff
@@ -224,3 +351,48 @@ func (t *terminaler) HandleSession(shell, namespace, podName, containerName stri
 
 	session.Close(1, "Process exited")
 }
+
+func (t *terminaler) HandleShellAccessToNode(nodename string, conn *websocket.Conn) {
+	succ, fail := make(chan bool), make(chan bool)
+
+	nodeTerminaler, err := NewNodeTerminaler(nodename, t.options, t.client)
+	if err != nil {
+		klog.Warning("node terminaler init error: ", err)
+		return
+	}
+
+	pod, err := nodeTerminaler.getNSEnterPod()
+	if err != nil {
+		klog.Warning("get nsenter pod error: ", err)
+		return
+	}
+
+	go nodeTerminaler.WatchPodStatusBeRunning(pod, succ, fail)
```

Comment:
there is a package in k8s `https://pkg.go.dev/k8s.io/apimachinery@v0.21.4/pkg/util/wait`, I think we can use `wait.Poll`.